ArjunaJTS/integration/src/main/java/com/arjuna/ats/jbossatx/jts/TransactionManagerService.java

```diff
@@ -92,7 +100,7 @@ public void start(org.omg.CORBA.ORB theCorbaORB) throws Exception
     public void stop() {
         super.stop();
 
-        ORB.getInstance("jboss-atx").shutdown();
+        ORB.getInstance(ORB_NAME).shutdown();
```

Comment:
The goal of the fix is:

- ensure that our shutdown hooks are called when the TM service is stopped
- ensure that we do not shutdown an externally supplied orb